test/instrumentations/amqplib/amqplibTestUtils.js

```diff
@@ -0,0 +1,69 @@
+import { SpanKind } from '@opentelemetry/api';
+
+export function getExpectedResourceAttributes() {
+  return {
+    'service.name': 'amqplib',
+    'telemetry.sdk.language': 'nodejs',
+    'telemetry.sdk.name': 'opentelemetry',
+    'telemetry.sdk.version': expect.any(String),
+    framework: 'node',
+    'process.environ': expect.any(String),
+    'lumigo.distro.version': expect.stringMatching(/1\.\d+\.\d+/),
+    'process.pid': expect.any(Number),
+    'process.executable.name': 'node',
+    'process.runtime.description': 'Node.js',
+    'process.runtime.name': 'nodejs',
+    'process.runtime.version': expect.stringMatching(/\d+\.\d+\.\d+/),
+  };
+}
+
+export function getExpectedSpan({
+  nameSpanAttr,
+  spanKind,
+  resourceAttributes,
+  host,
+  topic,
+  message,
+}) {
+  let messageKey = 'invalid-message-key';
+  switch (spanKind) {
+    case SpanKind.PRODUCER:
+      messageKey = 'messaging.publish.body';
+      break;
+    case SpanKind.CONSUMER:
+      messageKey = 'messaging.consume.body';
+      break;
+  }
```

Comment:
let's try to write more explicit code:
```suggestion
 if spanKind ==  SpanKind.PRODUCER
         messageKey = 'messaging.publish.body';
 else if spanKind == SpanKind.CONSUMER
        messageKey = 'messaging.consume.body';
 else
       assert(false);
```